gtfs/spec/en/reference.md

```diff
@@ -366,7 +369,31 @@ Describe the different levels of a station. Is mostly useful when used in conjun
 |  `level_index` | Float | **Required** | Numeric index of the level that indicates relative position of this level in relation to other levels (levels with higher indices are assumed to be located above levels with lower indices).<br><br>Ground level should have index 0, with levels above ground indicated by positive indices and levels below ground by negative indices.|
 |  `level_name` | Text | Optional | Optional name of the level (that matches level lettering/numbering used inside the building or the station). Is useful for elevator routing (e.g. “take the elevator to level “Mezzanine” or “Platforms” or “-1”).|
 
+### occupancies.txt
 
+File: **Optional**
+
+Usual or expected in-vehicle occupancy levels.
+
+For describing historical average occupancies at least 4 weeks of occupancy data are suggested to be averaged across each unique day of the week, valid for up to 2 weeks thereafter in `occupancies.txt`.
+
+Methods for occupancy forecasting used to populated `occupancies.txt` are not strictly specified. Beyond the guidelines given above, it is the responsibility of data producers to ensure that the data provided are meaningful. 
+
+| Field Name | Type | Required | Description |
+| ----- | ----- | ----- | ----- |
+| `trip_id` | ID referencing `stop_times.trip_id` | **Required** |  Identifies a `trip_id` for which an occupancy level is described. |
```

Comment:
I should not bring this up, but if this is a foreign key, then we must resolve the frequencies.txt issue.

Reply:
Indeed. The model for `occupancies.txt` seems to only support trips where `time_point=1`, and overlooks frequency-based trips (i.e., trips using `frequencies.txt`) entirely.

Its not clear how we would provide "stop time" level occupancies for frequency based trips since there is no single stop_time identifier. Perhaps by adding additional fields for "date" and "first vehicle journey departure time", but it gets a bit sketchy.

Reply:
What about adding start_time like we do with GTFS-RT?

Reply:
And start_date?

Reply:
Why would a start_data be relevant? It is not in gtfs-static...